### PR TITLE
refactor(connector): use @logto/connector-schemas to substitute @logto/connector-types

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

As we add @logto/connector-schemas, we should use this to substitute the connectors' implementation's dependency on @logto/connector-types.

## Testing

UT and IT.

### CODE REVIEW

File: packages/connector-alipay-native/src/index.ts [hunk 1]
```diff
@@ -174,6 +164,18 @@ export default class AlipayNativeConnector implements SocialConnectorInstance<Al
     return { id, avatar, name };
   };
 
+  protected readonly authResponseParser: AuthResponseParser<AuthResponse> = async (
```
Comment (wangsijie) [inline packages/connector-alipay-native/src/index.ts:167]: why not `private`

**Review by wangsijie — APPROVED** [review]
Overall LGTM

**Comment by wangsijie** [comment]
Better add some notes on why this PR exceeds 500 lines.

File: packages/connector-alipay-native/src/index.ts (hunk 1 above)
Comment (darcyYe) [reply packages/connector-alipay-native/src/index.ts:167]: it depends on the LogtoConnector design in another PR. It should be `protected` if this is defined in the parent class.

**Comment by wangsijie** [comment]
Per offline discussion, we will introduce a new design, close this.

**Pull request closed without merging**